proxy/HostStatus.h

```diff
@@ -41,6 +41,8 @@ enum HostStatus_t {
   HOST_STATUS_DOWN,
 };
 
+extern RecRawStatBlock *host_status_rsb;
```

Comment:
Don't expose this in the header unnecessarily. Make it static in the source file where it is used.